docs/sql-ref-syntax-qry-select-orderby.md

```diff
@@ -18,5 +18,121 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+The <code>ORDER BY</code> clause is used to return the result rows in a sorted manner
+in the user specified order. Unlike the <code>SORT BY</code> clause, this clause guarantees
+total order in the output. 
 
-**This page is under construction**
+### Syntax
+{% highlight sql %}
+ORDER BY { expression [ sort_direction | nulls_sort_oder ] [ , ...] }
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>ORDER BY</em></code></dt>
+  <dd>
+    Specifies a comma separated list of expressions along with optional parameters <code>sort_direction</code>
```

Comment:
```comma separated``` -> ```comma-separated```?
I caught this because I saw Sean's comment in another PR :)

Actually in many places we have ```comma separated```, for example, 
```
An optional parameter that specifies a comma separated list of key and value pairs for partitions.
```
Not sure if I should open a MINOR PR to fix all of them.